### PR TITLE
Create simple_Graph.js

### LINKED ISSUE #818: Graphs in Javascript
Can i start working on it ??

### PR DESCRIPTION
**Pull request by khushal87**
Fixes #818  

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Sample Input / Output.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:

- Adding simple_Graph.js
- 

#### Languages Used:
- Javascript
- 

#### Files Added:
- simple_Graph.js
-

### CODE REVIEW

File: Graphs-Data Structure/simple_Graph.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+class Graph {
```
Comment (jainaman224) [inline Graphs-Data Structure/simple_Graph.js]: Fix file and folder name.

Comment (khushal87) [reply Graphs-Data Structure/simple_Graph.js]: What will u name it to?

File: Graphs-Data Structure/simple_Graph.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    this.numberOfNodes++;
+  } 
+  
+  //to add edge
+  addEdge(node1, node2) { 
+    //uniderected Graph 
+    this.adjacentList[node1].push(node2); 
+    this.adjacentList[node2].push(node1); 
+  } 
+  //to show the connections
```
Comment (jainaman224) [inline Graphs-Data Structure/simple_Graph.js]: Add a new line above this.

File: Graphs-Data Structure/simple_Graph.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
+class Graph { 
+  constructor() {
```
Comment (jainaman224) [inline Graphs-Data Structure/simple_Graph.js]: Use 4 space indentation.

File: Graphs-Data Structure/simple_Graph.js (hunk 1 above)
Comment (jainaman224) [reply Graphs-Data Structure/simple_Graph.js]: What about Graph/Graph.js ?

**Comment by khushal87** [comment]
Done  @jainaman224

**Pull request merged**